packages/web-components/fast-components/src/slider/slider.styles.ts

```diff
@@ -101,63 +109,73 @@ export const sliderStyles: (
+        margin-top: calc((${designUnit} + calc(${density} + 5)) * 1px);
         height: calc(var(--track-width) * 1px);
     }
     :host([orientation="vertical"]) .track {
         top: calc(var(--track-overhang) * 1px);
         bottom: calc(var(--track-overhang) * 1px);
+        margin-inline-start: calc((${designUnit} + calc(${density} + 5)) * 1px);
         width: calc(var(--track-width) * 1px);
         height: 100%;
     }
     .track {
         background: ${neutralStrokeRest};
+        border: 1px solid ${neutralStrokeRest};
+        border-radius: 2px;
+        box-sizing: border-box;
         position: absolute;
-        border-radius: calc(${controlCornerRadius} * 1px);
     }
     :host([orientation="vertical"]) {
-        height: calc(var(--fast-slider-height) * 1px);
-        min-height: calc(var(--thumb-size) * 1px);
+        height: 100%;
+        min-height: calc(${designUnit} * 60px);
         min-width: calc(${designUnit} * 20px);
     }
-    :host([disabled]), :host([readonly]) {
+    :host([orientation="vertical"]) .track-start {
+        height: auto;
+        width: 100%;
+        top: 0;
+    }
+    :host(.disabled),
+    :host(.readonly) {
         cursor: ${disabledCursor};
     }
-    :host([disabled]) {
+    :host(.disabled) {
         opacity: ${disabledOpacity};
     }
-`.withBehaviors(
+    ::slotted([slot="thumb"]) {
+        color: ${neutralForegroundRest};
+        fill: currentcolor;
+    }
+  `.withBehaviors(
         forcedColorsStylesheetBehavior(
             css`
-            .thumb-cursor {
-                forced-color-adjust: none;
-                border-color: ${SystemColors.FieldText};
-                background: ${SystemColors.FieldText};
-            }
-            .thumb-cursor:hover,
-            .thumb-cursor:active {
-                background: ${SystemColors.Highlight};
-            }
-            .track {
-                forced-color-adjust: none;
-                background: ${SystemColors.FieldText};
-            }
-            :host(:${focusVisible}) .thumb-cursor {
-                border-color: ${SystemColors.Highlight};
-            }
-            :host([disabled]) {
-                opacity: 1;
-            }
-            :host([disabled]) .track,
-            :host([disabled]) .thumb-cursor {
-                forced-color-adjust: none;
-                background: ${SystemColors.GrayText};
-            }
-
-            :host(:${focusVisible}) .thumb-cursor {
-                background: ${SystemColors.Highlight};
-                border-color: ${SystemColors.Highlight};
-                box-shadow: 0 0 0 2px ${SystemColors.Field}, 0 0 0 4px ${SystemColors.FieldText};
-            }
-        `
+                .thumb-cursor {
+                    forced-color-adjust: none;
+                    border-color: ${SystemColors.FieldText};
+                    background: ${SystemColors.FieldText};
+                }
+                :host(:not(.disabled)) .thumb-cursor:hover,
```

Comment:
Should the class `.disabled` be used instead of the attribute selector `[disabled]`?

Reply:
updated to use `[disabled]`